### ISSUE TITLE
Unknown error in the indexing

### ISSUE THREAD

**Issue by jonaprieto** [algolia/jekyll-algolia#78]
I got this using the version 1.4.6 of jekyll-algolia.
It's weird because yestarday everything was working fine, I didn't touch any
essential. 

```
$ jekyll --version
jekyll 3.8.3
$ jekyll algolia
...
Rendering to HTML (100%) |============================================================================================|
jekyll 3.8.3 | Error:  undefined method `to_time' for "2018-07-03":String                                             |
Did you mean?  to_i
```

Any clue?

My repository still being http://github.com/jonaprieto/jonaprieto.github.io in the branch "sources"

**Comment by pixelastic** [comment]
Thanks for the report,

I did try on your website and was able to reproduce the issue. For a reason I don't explain, it seems that the dates of your collection items were passed as string (when they should have been converted to Date objects by Jekyll)

I didn't really understand how this happened (could not reproduce with a smaller repo test case), so I guess it was coming from another of your plugins.

I released v1.4.7 of jekyll-algolia that should gracefully handle this case now: it the date is passed as a string, it will try to convert it to a date object. You should be to have your build work again by upgrading to this version.

Thanks for the bug report!

**Comment by jonaprieto** [comment]
Indeed. It's working now. Thank you!
